internal/integration/cli/health.go

```diff
@@ -79,6 +82,13 @@ func (suite *HealthSuite) TestClientSideWithExplicitNodes() {
 //
 //nolint:gocyclo
 func (suite *HealthSuite) TestClientSideWithDiscovery() {
+	discoveryEnabled, err := suite.isDiscoveryEnabled()
+	suite.Require().NoError(err)
+
+	if !discoveryEnabled {
+		suite.T().Skip()
```

Comment:
one small nit here: we can use `Skipf()` with some meaningful message to indicate why we skipped to make it more obvious to someone looking at the test output

Reply:
makes sense, done.